wagtail/admin/templates/wagtailadmin/shared/side_panels/status.html

```diff
@@ -15,9 +15,9 @@
             {% endif %}
         </div>
         {% if object.first_published_at %}
-            <div class="w-help-text">
-                {% trans 'First published ' %}{% timesince_last_update object.first_published_at use_shorthand=True %}
-            </div>
-        {% endif %}
+    <div class="w-help-text">
+        {% trans 'First published' %} {% human_readable_date object.first_published_at position="bottom" %}
```

Comment:
Just a reminder to run `make format` when doing changes, this HTML has lost some of the nesting. I can fix this up though when merging.

Reply:
@lb- Thank you for the reminder! I'll make sure to run "make format" before submitting any changes in the future.